Libraries/LibJS/Bytecode/Instructions.h

```diff
@@ -0,0 +1,218 @@
+    X(GreaterThanEquals)                     \
+    X(HasPrivateId)                          \
+    X(ImportCall)                            \
+    X(In)                                    \
+    X(Increment)                             \
+    X(InitializeLexicalBinding)              \
+    X(InitializeVariableBinding)             \
+    X(InstanceOf)                            \
+    X(IteratorClose)                         \
+    X(IteratorNext)                          \
+    X(IteratorNextUnpack)                    \
+    X(IteratorToArray)                       \
+    X(LeaveFinally)                          \
+    X(LeaveLexicalEnvironment)               \
+    X(LeavePrivateEnvironment)               \
+    X(LeaveUnwindContext)                    \
+    X(LeftShift)                             \
+    X(LessThan)                              \
+    X(LessThanEquals)                        \
+    X(LooselyEquals)                         \
+    X(LooselyInequals)                       \
+    X(Mod)                                   \
+    X(Mul)                                   \
+    X(NewArray)                              \
+    X(NewClass)                              \
+    X(NewFunction)                           \
+    X(NewObject)                             \
+    X(NewPrimitiveArray)                     \
+    X(NewRegExp)                             \
+    X(NewTypeError)                          \
+    X(Not)                                   \
+    X(PrepareYield)                          \
+    X(PostfixDecrement)                      \
+    X(PostfixIncrement)                      \
+    X(PutBySpread)                           \
+    X(PutPrivateById)                        \
+    X(ResolveSuperBase)                      \
+    X(ResolveThisBinding)                    \
+    X(RestoreScheduledJump)                  \
+    X(RightShift)                            \
+    X(SetCompletionType)                     \
+    X(SetGlobal)                             \
+    X(SetLexicalBinding)                     \
+    X(SetVariableBinding)                    \
+    X(StrictlyEquals)                        \
+    X(StrictlyInequals)                      \
+    X(Sub)                                   \
+    X(SuperCallWithArgumentArray)            \
+    X(Throw)                                 \
+    X(ThrowIfNotObject)                      \
+    X(ThrowIfNullish)                        \
+    X(ThrowIfTDZ)                            \
+    X(Typeof)                                \
+    X(TypeofBinding)                         \
+    X(UnaryMinus)                            \
+    X(UnaryPlus)                             \
+    X(UnsignedRightShift)
+
+#define JS_ENUMERATE_SIMPLE_JUMP_OPS(X) \
+    X(Jump)                             \
+    X(JumpIf)                           \
+    X(JumpTrue)                         \
+    X(JumpFalse)                        \
+    X(JumpNullish)                      \
+    X(JumpUndefined)
```

Comment:
Didn't we have a different copy of this somewhere? please define that in terms of this instead.

Reply:
Right, so move these there and define those in terms of these :)